.storybook/main.ts

```diff
@@ -71,20 +73,17 @@ const webpackConfig = (config) => {
 }
 
 module.exports = {
-  core: {
-    builder: 'webpack5',
-  },
   stories: ['../src/**/*.stories.@(ts|tsx)'],
   addons: ['@storybook/addon-essentials', '@storybook/addon-a11y'],
-  typescript: {
-    check: false,
-    checkOptions: {},
-    reactDocgen: 'react-docgen-typescript',
-    reactDocgenTypescriptOptions: {
-      tsconfigPath: path.resolve(__dirname, '../tsconfig.json'),
-    },
-  },
```

Comment:
Question: Why was this section removed?

Reply:
An incorrect assumption that storybook default settings covered this, but I failed to notice the disabling of the check property. This section has been returned.